etna/articles/models.py

```diff
@@ -72,11 +73,40 @@ def get_context(self, request):
 @register_snippet
 class ArticleTag(TagBase):
     free_tagging = False
+    skos_id = models.CharField(
+        blank=True,
+        unique=True,
+        db_index=True,
+        max_length=100,
+        verbose_name="SKOS identifier",
+        help_text="Used as the identifier for this tag when sending page metatdata to the CIIM API.",
```

Comment:
Question: 
1) What does an skos identifer? Is it something to do with "a common SKOS reference exists for certain taxonomies"?
2) what is `skos_id` verbose text than the same `id` of that model?

Reply:
@TNA-Allan Did you take a look at the ticket and the attached document on there?

Reply:
Yes, I did, I tried to map it with the structure to see if there is a `skos_id` to it, but not. Perhaps a chat on this?

Reply:
Thanks for the chat @ababic